src/sentry/static/sentry/app/components/events/interfaces/exceptionStacktraceContent.tsx

```diff
@@ -46,11 +49,23 @@ const ExceptionStacktraceContent = ({
     );
   }
 
+  /**
+   * Armin:
+   * If all frames are in app, then no frame is in app.
+   * This normally does not matter for the UI but when chained exceptions
+   * are used this causes weird behavior where one exception appears to not have a stacktrace.
+   *
+   * It is easier to fix the UI logic to show a non-empty stacktrace for chained exceptions
```

Comment:
I think _I_ posted that in slack btw, not armin

Reply:
Oh, the top sentences were copy-pasted from Armin's asana comment, but you are right, the last one was yours - I'll add the mention so that we don't loose reference to the original authors.